online_meetings/bluejeans_report.py

```diff
@@ -0,0 +1,186 @@
+import requests
+# standard libraries
+import json
+import logging
+import os
+import sys
+import yaml
+
+# third-party libraries
+import pandas as pd
+import furl
+import time
+import dateparser
+import pytz
+from datetime import datetime, timedelta
+import glob
+
+# importing required modules
+from zipfile import ZipFile
+
+# Initialize settings and globals
+
+logger = logging.getLogger(__name__)
+
+# Default time to sleep when hitting a 429 limit
+DEFAULT_SLEEP_TIME = 10
+
+# read configurations
+try:
+    with open(os.path.join(os.path.dirname(__file__), '../config/env.json')) as env_file:
```

Comment:
Sam said the path for this is ../config/secrets/env.json.

This will be updated in a central manager but we might as well just put all our files in the same place.